### ISSUE TITLE
RaggedTensor requires merge_dims method

### ISSUE THREAD

**Issue by nicain** [tensorflow/text#142]
On this line:
https://github.com/tensorflow/text/blob/15ffafec84080d1c0accd7d4d4be17dd5497d299/tensorflow_text/python/ops/bert_tokenizer.py#L125

RaggedTensor uses the merge_dims method.  However, this feature is currently not deployed in tf 2.0; it looks like it was added here: https://github.com/tensorflow/tensorflow/commit/6f29f5cbd16d65db399b96e956e6cd0a9ab8555c

but has yet to be deployed.

Minimal reproducing example:
```python
import tensorflow as tf
import tensorflow_text as tft

num_oov_buckets = 3
initializer = tf.lookup.KeyValueTensorInitializer(['Hello', 'World'], [0, 1], value_dtype=tf.int64)
table = tf.lookup.StaticVocabularyTable(initializer, num_oov_buckets)

data = tf.constant(['World Hello missing'])
tok = tft.BertTokenizer(table)

session = tf.compat.v1.Session()
tf.compat.v1.tables_initializer()

output = tok.tokenize(data)

print(output[0][0].eval(session=session))
print(output[0][1].eval(session=session))
print(output[0][2].eval(session=session))
```

Traceback:
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/tensorflow_text/python/ops/bert_tokenizer.py", line 195, in tokenize
    tokens = self._basic_tokenizer.tokenize(text_input)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/tensorflow_text/python/ops/bert_tokenizer.py", line 126, in tokenize
    return final_tokens.merge_dims(-2, -1)
AttributeError: 'RaggedTensor' object has no attribute 'merge_dims'

**Comment by thuang513** [comment]
Thanks for reporting this nicain! I have a change in the works that will fix this and converge some differences we discovered between the original bert tokenizer and this tokenizer. Stay tuned!

**Comment by thuang513** [comment]
This should be fixed in commit 3be9f9ec70458c337d64ce5b7164f88f814f4041
